frame/whitelist/src/lib.rs

```diff
@@ -80,8 +81,7 @@ pub mod pallet {
 		type DispatchWhitelistedOrigin: EnsureOrigin<Self::RuntimeOrigin>;
 
 		/// The handler of pre-images.
-		// NOTE: recipient is only needed for benchmarks.
-		type PreimageProvider: PreimageProvider<Self::Hash> + PreimageRecipient<Self::Hash>;
```

Comment:
Every rename is a potential annoyance for para-chains. Just saying.

Reply:
Considered it, since the pallet is new and probably used only in kusama runtime, I chose a consistency.
It's a way it's named in other pallets.